src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,57 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  query: string;
+  setQuery: (value: string) => void;
+  selectFilter: string;
+  setSelectFilter: (option: string) => void;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  setQuery,
+  selectFilter,
+  setSelectFilter,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            value={selectFilter}
+            onChange={e => setSelectFilter(e.target.value)}
```

Comment:
Consider adding a type annotation for the event parameter in the `onChange` handler for better type safety. You can use `React.ChangeEvent<HTMLSelectElement>` for the select element.